README.md

```diff
@@ -28,259 +28,35 @@ different networks, you can read more about this in [this article](https://tezos
-sudo apt-get install software-properties-common
-# Add PPA with Tezos binaries
-sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
-sudo apt-get update
-```
-
-And install packages with `apt-get`, e.g. for `tezos-client`:
-```
-sudo apt-get install tezos-client
-```
-
-<a name="fedora"></a>
-## Fedora Copr repository with `tezos-*` binaries
-
-If you are using Fedora you can use Copr in order to install `tezos-*`
-executables.
-E.g. in order to install `tezos-client` or `tezos-baker` run the following commands:
-```
-# use dnf
-sudo dnf copr enable @Serokell/Tezos
-sudo dnf install tezos-client
-sudo dnf install tezos-baker-010-PtGRANAD
-
-# or use yum
-sudo yum copr enable @Serokell/Tezos
-sudo yum install tezos-baker-010-PtGRANAD
-```
-Once you install such packages the commands `tezos-*` will be available.
-
-<a name="linux"></a>
-## Other Linux distros usage
-
-Download binaries from release assets.
-
-### `tezos-client` example
-
-Make it executable:
-```
-chmod +x tezos-client
-```
-
-Run `./tezos-client` or add it to your PATH to be able to run it anywhere.
-
-<a name="macos"></a>
-## Brew tap for macOS
-
-If you're using macOS and `brew`, you can install Tezos binaries from the taps provided
-by this repository. There are two taps: one for the latest stable release of Tezos and
-one for the latest release candidate of Tezos.
-
-In order to use latest stable version run the following:
-```
-brew tap serokell/tezos-packaging-stable https://github.com/serokell/tezos-packaging-stable.git
-```
-
-In order to use latest release candidate version run the following:
-```
-brew tap serokell/tezos-packaging-rc https://github.com/serokell/tezos-packaging-rc.git
-```
-
-Once the desired tap is selected, you can install the chosen package, e.g.:
-```
-brew install tezos-client
-```
-
-### Building brew bottles
-
-It's possible to provide prebuilt macOS packages for brew called bottles. They're supposed
-to be built before making the new release and included to it. In order to build all bottles run
-`build-bottles.sh` script:
-```
-./scripts/build-bottles.sh
-```
-
-Note that this might take a while, because builds don't share common parts and for each binary
-dependencies are compiled from scratch. Once the bottles are built, the corresponding sections in the
-formulas should be updated. Also, bottles should be uploaded to the release artifacts.
+  This option also supports **Raspberry Pi OS**, see [this part of the doc](./docs/distros/ubuntu.md#raspberry).
+* **Fedora**. Install packages using `dnf` or `yum` following [these instructions](./docs/distros/fedora.md).
+* Other linux distros can use static binaries from the [latest release](https://github.com/serokell/tezos-packaging/releases/latest).
+* **macOS**. Install formulaes using `brew` following [these instructions](./docs/distros/macos.md).
```

Comment:
A mostly aesthetic suggestion I'd make is to make a heading for each entry, which is itself the link, and only put a small description below it.

Here my suggestion for this whole section would be:
```
<a name="getting-binaries"></a>
## Getting Tezos binaries

The following distributions are supported by the `tezos-packaging`:

### [macOS](./docs/distros/macos.md)

`brew` formulae and taps for stable releases and release candidates.

### [Ubuntu](./docs/distros/ubuntu.md)

Native Ubuntu packages that can be installed with `apt-get` from a PPA.

### [Rasperry Pi OS](./docs/distros/ubuntu.md#raspberry)

On some Raspberry systems Ubuntu packages can be installed using `apt-get`.

### [Fedora](./docs/distros/fedora.md)

Native Fedora packages that can be installed using using `dnf` or `yum`.

### [Static binaries and other linux distros](https://github.com/serokell/tezos-packaging/releases)

Prebuilt static binaries can be downloaded directly from the [latest release](https://github.com/serokell/tezos-packaging/releases/latest).
```

I've see this done in some other READMEs and I think it can be easier to read.